lib/pavilion/plugins/commands/wait.py

```diff
@@ -60,6 +70,19 @@ def run(self, pav_cfg, args):
 
             tmp_statuses = status.get_statuses(pav_cfg, args, self.errfile)
 
+            # print status every 5 seconds
```

Comment:
Why not use the status command bits to just print the status table as normal?

Reply:
It will reprint the table each time, headers included:

```
 Test statuses                                                   
---------+---------------+---------+----------+--------------------
 Test id | Name          | State   | Time     | Note               
---------+---------------+---------+----------+--------------------
 11      | test.basic | RUNNING | 16:49:57 | Currently running. 

 Test statuses                                                   
---------+---------------+---------+----------+--------------------
 Test id | Name          | State   | Time     | Note               
---------+---------------+---------+----------+--------------------
 11      | test.basic | RUNNING | 16:49:57 | Currently running. 
```